### ISSUE TITLE
Please allow overriding signtool.exe with custom signing executable

### ISSUE THREAD

**Issue by theresia-tobii** [clowd/Clowd.Squirrel#29]
We need to be using a different tool for signing (we have one in our company that signs via azure). It would be great if there was an argument to squirrel.exe to be able to use it in addition to the signParams that are already there. Something like this (applied to the sample from the docs):

```
Squirrel pack`
 --framework net6,vcredist143-x86`  # Install .NET 6.0 (x64) and vcredist143 (x86) during setup, if not installed
 --packName "YourApp"`              # Application / package name
 --packVersion "1.0.0"`             # Version to build. Should be supplied by your CI
 --packAuthors "YourCompany"`       # Your name, or your company name
 --packDirectory ".\publish"`       # The directory the application was published to
 --setupIcon "mySetupIcon.ico"`     # Icon for Setup.exe
 --splashImage "install.gif"        # The splash artwork (or animation) to be shown during install
 --signingExe "C:/path/to/my/signtool.exe"
 --signParams "params to my signtool"
```

This should be calling C:/path/to/my/signtoo.exe params to my signtool (note that the "sign" argument should not be in here if overriding the signingExe).

For reference, ours needs to be called like this: AzureSignTool.exe -f "filetosign" -p "password"

**Comment by caesay** [comment]
This is a good idea, I will work on this for the next prerelease.

**Comment by caesay** [comment]
Just a question, from looking at the documentation on AzureSignTool.exe, it appears to follow the same command line convention as the regular signtool.exe.. https://github.com/vcsjones/AzureSignTool

Do we need to drop the 'sign' keyword and add the `-f` parameter? if that's true (particularly the extra parameter) this becomes far more complex, because we might also need to add a `--signToolFileParam` argument etc.

**Comment by caesay** [comment]
A simple version of this is now [in 2.7.79](https://github.com/clowd/Clowd.Squirrel/releases/tag/2.7.79-pre). Please let me know if this works for you or if there are additional changes needed.

**Comment by theresia-tobii** [comment]
The azuresigntool.exe that we have is one of our company's own making.

**Comment by theresia-tobii** [comment]
I will have a go and see if the "sign" in there breaks things in our use case or not.

A more generically applicable option could perhaps be to allow something along the following in `--signCommand "path/to/OurAzureSignTool.exe -p password -f {0}"` where the file would be placed in {0} by the means of string.Format or something similar.

**Comment by caesay** [comment]
Commit edda9c8e6ecd3df46de0642baf511064c8fc1afe adds support for a completely custom signing command, it will be in the next pre-release.

**Comment by theresia-tobii** [comment]
Wow, that was quick. Thanks.

**Comment by caesay** [comment]
This has been released [in 2.7.89-pre](https://github.com/clowd/Clowd.Squirrel/releases/tag/2.7.89-pre).
